### PR TITLE
New: Add click event trigger (solves: #106)

### PR DESCRIPTION
**Pull request by joe-replin**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### New
* Adds event trigger for the purpose of tracking user selection of resources
* Solves for #106 

[//]: # (List appropriate steps for testing if needed)
### Testing
1. Install this PR
2. Listen to event trigger of resources and ensure item data is pulling through.

[//]: # (Mention any other dependencies)

### CODE REVIEW

File: js/ResourcesView.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default class ResourcesView extends Backbone.View {
   }
 
   render() {
+    this.onClick = this.onClick.bind(this);
```
Comment (oliverfoster) [inline js/ResourcesView.js:18]: Move this to initialize

File: js/ResourcesView.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ export default class ResourcesView extends Backbone.View {
   initialize() {
     this.listenTo(Adapt, 'remove', this.remove);
     this.render();
+
+    this.onResourcesItemClicked = this.onResourcesItemClicked.bind(this);
```
Comment (oliverfoster) [inline js/ResourcesView.js:16]: These should occur befor everything else in the initialise function, as often function binds are used in listenTo and render functions, as in this case.

Comment (joe-replin) [reply js/ResourcesView.js]: Done! Thank you.

File: js/ResourcesView.js (hunk 1 above)
Comment (joe-replin) [reply js/ResourcesView.js]: Done! Thank you.

**Review by oliverfoster — APPROVED** [review]

**Review by swashbuck — APPROVED** [review]
👍

**Pull request merged**